modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -65,11 +65,7 @@ boolean limitToOneSubmissionPerUser = DDMFormInstanceSubmissionLimitStatusUtil.i
 		<%
 		DDMFormInstance formInstance = ddmFormDisplayContext.getFormInstance();
 
-		boolean expired = false;
-
-		if (ddmFormDisplayContext.isExpirationDateEnabled()) {
-			expired = DDMFormInstanceExpirationStatusUtil.isFormExpired(formInstance, timeZone);
-		}
+		boolean expired = DDMFormInstanceExpirationStatusUtil.isFormExpired(formInstance, timeZone);
 
```

Comment:
Blank line is not needed.

Reply:
Hi paulino! The SF said it was needed so that's why its there

Reply:
Oh right. The `formInstance` is declared right before! Thanks for letting me know, @rebsilva